lib/src/Uncertainty/Distribution/KernelMixture.cxx

```diff
@@ -661,6 +661,55 @@ KernelMixture::PointWithDescriptionCollection KernelMixture::getParametersCollec
   return parameters;
 } // getParametersCollection
 
+
+Point KernelMixture::getParameter() const
+{
+  const UnsignedInteger dimension = getDimension();
+  const UnsignedInteger size = sample_.getSize();
+  Point parameter;
+  for (UnsignedInteger i = 0; i < dimension; ++ i)
+  {
+    for (UnsignedInteger j = 0; j < size; ++ j)
+    {
+      parameter.add(sample_[j][i]);
```

Comment:
For performance reasons, always write `sample_(j, i)` instead of `sample_[j][i]`.  Same thing in setParameter.

Reply:
Ok, I guess it doesnt matter if the order is not consistent with getParametersCollection